### ISSUE TITLE
Wrong Validation Loss with Multi-Output Architecture

### ISSUE THREAD

**Issue by xv1nx** [keras-team/keras#13601]
**System information**  
- TensorFlow backend (yes / no):   yes
- TensorFlow version:  1.13.2
- Keras version:  2.3.1
- Python version:  3.6
- CUDA/cuDNN version:  10.0

**Describe the current behavior**  
While training a network with multiple output layers (3 in my case), i observe wrong validation loss:
`Epoch 20/100`
`1281/1281 [==============================] - 1549s 1s/step - loss: 219.5982 - l_out_loss: 49.2567 - r_out_loss: 42.5549 - s_out_loss: 110.9972 - val_loss: 1.7971 - val_l_out_loss: 13944.2723 - val_r_out_loss: 20445.2947 - val_s_out_loss: 9993.8637`
**Describe the expected behavior**  
The validation loss should look like the one of the training. Even for the training, the loss is not the sum of all outputs, but it has the same dimension.
I don't have this problem with keras 2.2.4. This Problem pop up from version 2.2.5 upwards. Unfortunately, i can't go back to 2.2.4, because I need the new on_test_begin callback.
**Code to reproduce the issue**  
Provide a reproducible test case that is the bare minimum necessary to generate the problem.  

**Other info / logs**  

`mod.compile(optimizer, loss=[myLossE1, myLossE2, myLossE3])`
`mod.fit(x_train, [y_train, y_train, y_train],
            steps_per_epoch=len(train_dataset) // train_dataset.batch_size,
            epochs=1000,
            validation_data=(x_val, [y_val, y_val, y_val]),
            validation_steps=len(validation_dataset) // validation_dataset.batch_size,
            callbacks=callbacks_list)`
